CondTools/SiStrip/test/SiStripApvSimulationParametersBuilder_cfg.py

```diff
@@ -27,19 +27,31 @@
     apvBaselines_minBaseline=cms.double(0.),
     apvBaselines_maxBaseline=cms.double(738.),
     apvBaselines_puBinEdges=cms.vdouble(0.,  2.,  4.,  6.,  8., 10., 12., 14., 16., 18., 20., 22., 24., 26., 28., 30., 32., 34., 36., 38., 40., 42., 44., 46., 48., 50.),
-    apvBaselines_zBinEdges=cms.vdouble(0., 25., 50., 75.),
-    apvBaselines_rBinEdges_TID = cms.untracked.vdouble(),
-    apvBaselines_rBinEdges_TEC = cms.untracked.vdouble(),
-    apvBaselinesFile_tib1=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TIB1_11us.txt"),
-    apvBaselinesFile_tib2=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TIB2_14us.txt"),
-    apvBaselinesFile_tib3=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TIB3_15us.txt"),
-    apvBaselinesFile_tib4=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TIB4_17us.txt"),
-    apvBaselinesFile_tob1=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TOB1_9us.txt"),
-    apvBaselinesFile_tob2=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TOB2_11us.txt"),
-    apvBaselinesFile_tob3=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TOB3_14us.txt"),
-    apvBaselinesFile_tob4=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TOB4_17us.txt"),
-    apvBaselinesFile_tob5=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TOB5_21us.txt"),
-    apvBaselinesFile_tob6=cms.untracked.FileInPath("SimTracker/SiStripDigitizer/data/APVBaselines_TOB6_21us.txt")
```

Comment:
@EmyrClement I made these parameters untracked in https://github.com/cms-sw/cmssw/pull/28024. 
Can you update it it here as well?

Reply:
@EmyrClement @mmusich could you please clarify the rationale behind having all these parameters ```untracked```? Aren't they definiting the physics behaviour of the code?

Reply:
@fabiocos, no. The `ESproducer` is used only to produce an sqlite file which is then used to be uploaded to the condition DB and is read back into cmssw via the `GlobalTag` `ESSource`, so it does not affect physics. In case in the future one would like to use the `ESProducer` directly to fill the `EventSetup` these could be made tracked again, but it's not the current flow.

Reply:
@mmusich ok, I would say that they are defining a physics behaviour but in an auxiliary code, not directly in production, therefore their being tracked is to a good extent irrelevant so far